build/Query.js

```diff
@@ -220,7 +220,7 @@ class Query {
     parts.push({
       text: {
         query: terms,
-        path: ['paragraphs', 'code.lang'],
+        path: ['paragraphs', 'code.lang', 'code.value', 'text'],
```

Comment:
Preliminary testing with code.value being boosted significantly was causing weird code blocks to bubble up aggressively when searching for common parameters like `$and` or `$ne` so am relegating it to the same as content fields.